playbooks/02 - Use Case - File Content Extraction/> Hunt Indicators on Qradar.json

```diff
@@ -0,0 +1,356 @@
+{
+    "@type": "Workflow",
+    "triggerLimit": null,
+    "name": "> Hunt Indicators on Qradar",
+    "aliasName": null,
+    "tag": null,
+    "description": "Hunt indicator value on FortiSIEM for the past X Minutes",
```

Comment:
FortiSIEM is expected here?

Reply:
Qradar so far, if anyone needs it in the future it can be added

Reply:
Should we change description to Qradar then? @ftnt-cse

Reply:
the idea is to have several playbooks (one for each SIEM) in the feature and use it in a similar way to the modular enrichment framework we have, so SIEM is probably a better name for it